src/main/scala/com/databricks/spark/sql/perf/Benchmark.scala

```diff
@@ -597,7 +597,7 @@ abstract class Benchmark(
         // to scala.
         // The executionTime for the entire query includes the time of type conversion
         // from catalyst to scala.
-        var result: Option[Long] = None
+        var result: Option[java.lang.Long] = None
```

Comment:
Its a little odd for this be both an `Option` and a boxed `java.lang.Long`.  Can we pick one or the other?  Having a `Some(null)` is really confusing.